### PR TITLE
Fixup jest tests

### LINKED ISSUE #1502: Javascript tests don't work
16 of 16 tests fail, however the build is not marked as failed.  Should we just delete these or can we get them working again? Can we make it so that the result of the javascript test impacts the overall build status. 

all have this failure:
```
FAIL spec/javascripts/no_js_spec.js
  ● Test suite failed to run

    TypeError: Cannot redefine property: window

      10 |
      11 | global.readFixtures = readFixtures;
    > 12 | global.window = window;
         |              ^
      13 | global.jQuery = $;
      14 | global.$ = global.jQuery;
      15 | global.List = List;

      at Object.<anonymous> (setupJest.js:12:14)
```

### PR DESCRIPTION
**Pull request by marlo-longley**
Fixes #1502

If I change a JS test to fail, the build/suite falis on `bundle exec rake`

```sh
sul-requests git:(jstest) ✗ bundle exec rake

... [ the ruby tests ] ...

Finished in 1 minute 16.61 seconds (files took 5.53 seconds to load)
1047 examples, 0 failures, 11 pending

Coverage report generated for RSpec to /Users/marloelilongley/Projects/sul-requests/coverage. 8694 / 8992 LOC (96.69%) covered.
yarn run v1.22.17
$ jest -c jest.config.js
Getting asset pipeline lookup path from Rails
Getting asset pipeline lookup path from Rails
 PASS  spec/javascripts/item_selector_limit_message_spec.js (19.911 s)
 FAIL  spec/javascripts/toggle_show_tabs_spec.js (20 s)
  ● Toggling show tabs › addToggleBehavior() › it allows the user to toggle back and forth between panes

    expect(received).toBeVisible()

    received value must be an HTMLElement or an SVGElement.
    Received has value: null

      12 |       toggleShowTabs.addToggleBehavior();
      13 |
    > 14 |       expect(document.querySelector('#thing-x')).toBeVisible();
         |                                                  ^
      15 |       expect(document.querySelector('#thing-b')).not.toBeVisible();
      16 |       
      17 |       $('#thing-a a').click();

      at __EXTERNAL_MATCHER_TRAP__ (node_modules/expect/build/index.js:342:30)
      at Object.toBeVisible (node_modules/expect/build/index.js:343:15)
      at Object.<anonymous> (spec/javascripts/toggle_show_tabs_spec.js:14:50)

 PASS  spec/javascripts/item_selector_filtering_spec.js (19.998 s)
 PASS  spec/javascripts/item_approval_spec.js (19.951 s)
 PASS  spec/javascripts/item_selector_spec.js (19.964 s)
 PASS  spec/javascripts/no_js_spec.js (20.029 s)
 PASS  spec/javascripts/item_selector_breadcrumbs_spec.js (20.065 s)
 PASS  spec/javascripts/additional_user_validation_fields_spec.js (20.038 s)
 PASS  spec/javascripts/item_selector_zero_message_spec.js (20.093 s)
 PASS  spec/javascripts/item_selector_ad_hoc_items_spec.js (20.107 s)
 PASS  spec/javascripts/date_selector_spec.js (20.046 s)
 PASS  spec/javascripts/item_selector_active_spec.js (20.052 s)
 PASS  spec/javascripts/paging_schedule_updater_spec.js (20.071 s)
 PASS  spec/javascripts/item_selector_limit_spec.js (20.114 s)
 PASS  spec/javascripts/mediation_table_spec.js (20.156 s)
 PASS  spec/javascripts/item_selector_incrementor_spec.js
 PASS  spec/javascripts/item_selector_checkedout_note_spec.js

Test Suites: 1 failed, 16 passed, 17 total
Tests:       1 failed, 69 passed, 70 total
Snapshots:   0 total
Time:        22.759 s, estimated 25 s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
JavaScript tests failed





```

### CODE REVIEW

**Comment by marlo-longley** [comment]
I don't know how to get rid of : `tests (3.2) Expected — Waiting for status to be reported`

**Comment by cbeer** [comment]
Do you need to use a matrix for the node version? The default name for the test (used for github protected branch) derives from the matrix options... but I think it'd be safe to hard-code it in the setup node job?

**Comment by marlo-longley** [comment]
@cbeer I changed that -- but now it's hanging with that cached message again since it has the old name??

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -24,6 +24,11 @@ jobs:
         key: ${{ runner.os }}-gems-202103-${{ hashFiles('**/Gemfile.lock') }}
         restore-keys: |
           ${{ runner.os }}-gems-202103
+    - name: Use Node.js 18.15
+      uses: actions/setup-node@v3
+      with:
+        # keep pinned to 18.15 until we upgrade past node 18 to avoid this error in CI: https://github.com/nodejs/node/issues/47563
+        node-version: ['18.15']
```
Comment (cbeer) [inline .github/workflows/ruby.yml:31]:
```suggestion
        node-version: '18.15'
```

**Pull request merged**